pandas/core/frame.py

```diff
@@ -8564,7 +8564,8 @@ def pivot(self, index=None, columns=None, values=None) -> DataFrame:
         margins : bool, default False
             Add all row / columns (e.g. for subtotal / grand totals).
         dropna : bool, default True
-            Do not include columns whose entries are all NaN.
+            Do not include columns whose entries are all NaN. If true:
+            exclude rows containing missing data from aggregated margins.
```

Comment:
I think this is making things more confusing. I find just having the original description clearer. What is the problem you're trying to address here?

Reply:
I want to clarify for pviot_table, if any column within a row contains NaN, the entire row won't be included when aggregating margins. Currently, users expect values within the row not NaN to be aggregated.

Reply:
I see. I don't think the description you added is veey clear, at least it wasn't for me. Something like `When this value is True, rows with a NaN value in any column will be discarded before computing margins.` maybe? Or maybe you can find something better.